src/ccontrol/MergingHandler.cc

```diff
@@ -295,9 +289,6 @@ qdisp::MergeEndStatus MergingHandler::flushHttp(string const& fileUrl, uint64_t
          "MergingHandler::" << __func__ << " uberJob=" << uberJob->getIdStr() << " fileUrl=" << fileUrl);
 
     qdisp::MergeEndStatus mergeStatus = _mergeHttp(uberJob, fileUrl, fileSize);
-    if (mergeStatus.success) {
```

Comment:
Curious: was this found redundant / vestigial / potentially problematic / now handled somewhere else?

Reply:
It's now handled  directly in `InfileMerger::mergeHttp` as `_infileMerger->mergeCompleteFor(...)` just added bytes to total byte counts. Things moved around a bit, so the total can be added once the merge finishes in `InfileMerger::mergeHttp`